include/sympack/symPACKMatrix2D.hpp

```diff
@@ -651,24 +726,31 @@ namespace symPACK{
         }
 
         // Copy constructor.  
+        // TODO: Edit this with new fields
         blockCell_t ( const blockCell_t & other ): blockCell_t() {
+#ifdef CUDA_MODE
+          UPCXX_ASSERT(!_d_nzval && !is_gpu_block, "This function is unimplemented for blocks using host-bypass communication");
```

Comment:
Sorry, I of course meant all these assertions to be on the source object, sorry to inadvertently mislead you.
```suggestion
          UPCXX_ASSERT(!other._d_nzval && !other.is_gpu_block, "This function is unimplemented for blocks using host-bypass communication");
```

Reply:
fixed in [26f2b7d](https://github.com/symPACK/symPACK/pull/12/commits/26f2b7d9f549cb2b1a6fe3a17af42076c047452f)